torch/distributed/_fsdp/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List
+
+from .fully_sharded_data_parallel import FullyShardedDataParallel
```

Comment:
Can you please elaborate why this is necessary as `__all__` is empty?

Reply:
will remove __all__